static/js/compose.js

```diff
@@ -538,6 +544,12 @@ exports.transmit_message = function (request, success, error) {
         send_message_ajax(request, success, error);
     }
 };
+function report_content_not_sent(request) {
+    var message_undelivered_id = "" + request.local_id;
+    message_undelivered_id = "div[zid='" + message_undelivered_id.replace( /(:|\.|\[|\]|,|=|@)/g, "\\$1" ) + "']";
```

Comment:
I really doubt this regex is necessary, but if it is, you should add a comment in the code explaining typical incoming and outgoing values and what the regex does.